examples/hyperparamopt/hpo_example.py

```diff
@@ -0,0 +1,130 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Example for using hyperparameter optimization (hpo) package.
+
+In this example, we will try to optimize a function of
+2 variables (branin) using both hpo and grid search. 
+
+"""
+
+import brainiak.hyperparamopt.hpo as hpo
+import numpy as np
+import scipy.stats as st
+import matplotlib.pyplot as plt
+
+# Branin is the function we want to minimize.
+# It is a function of 2 variables
+def branin(x1, x2):
```

Comment:
Does branin mean something? Is it someone's name? Why is this a good loss function for us?